packages/pwa-kit-runtime/src/utils/ssr-namespace-paths.js

```diff
@@ -5,39 +5,40 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
+import {getConfig} from './ssr-config'
+
 /**
  * This file defines the /mobify paths used to set up our Express endpoints.
  *
  * If a namespace for the /mobify paths is defined, the methods in here will return the
  * namespaced path. ie. /namespace/mobify/...
  */
 
+// The MOBIFY_PATH is defined separately in preparation for the future eventual removal or
+// replacement of the 'mobify' part of these paths
 const MOBIFY_PATH = '/mobify'
 const PROXY_PATH_BASE = `${MOBIFY_PATH}/proxy`
 const BUNDLE_PATH_BASE = `${MOBIFY_PATH}/bundle`
 const CACHING_PATH_BASE = `${MOBIFY_PATH}/caching`
 const HEALTHCHECK_PATH = `${MOBIFY_PATH}/ping`
 const SLAS_PRIVATE_CLIENT_PROXY_PATH = `${MOBIFY_PATH}/slas/private`
 
-/**
- * @private
- */
-const _getNamespace = () => {
-    // TODO - namespaces for /mobify path will be implemented at a later date.
-    // Returns an empty string for now.
-    // Below is an example of what this implementation might look like.
-    /*
-        let {namespace = ""} = getConfig()
-        namespace = typeof namespace === 'function' ? namespace() : namespace
-        return namespace
-    */
-    return ''
+// TODO - Allow projects to define this function?
+export const getEnvBasePath = () => {
```

Comment:
getConfig is memoized on the server side so there should not be much of a performance impact there.
I think we are also ok client side since there we are just reading from `window.__config__` once it's hydrated.

I removed the note about allowing projects to define this. I think for now we will keep this function definition internal.